test/domain/CDSCashStatementsByMonthSpec.scala

```diff
@@ -24,7 +24,7 @@ import utils.{MustMatchers, SpecBase}
 
 import java.time.LocalDate
 
-class CashStatementsByMonthSpec extends SpecBase with MustMatchers {
+class CDSCashStatementsByMonthSpec extends SpecBase with MustMatchers {
```

Comment:
(hoping it is not on Purpose) This spec (file and class name) should be reverted to earlier names as the class name (this spec is for) is CashStatementsByMonth .

 If this is on purpose then the class name (this spec if for) must also be renamed